core/platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/DefaultWorkspaceConfigValidator.java

```diff
@@ -116,4 +127,14 @@ private void requiredNotNull(Object object, String message) throws BadRequestExc
             throw new BadRequestException(message);
         }
     }
+
+    /**
+     * Checks that first argument is not null nor empty string, throws {@link BadRequestException}
+     * in the case of null or empty {@code string} with given {@code message}.
+     */
+    private void requiredNeitherNullNorEmpty(String string, String message) throws BadRequestException {
```

Comment:
I don't think it is a good to make such checks.
The motivation is that what if we will need another check e.g. _non-null and non-empty collection_ should we then add a new method? My proposal is to change this method to something like `checkArgument(boolean condition, String message)` and reuse it throughout the validator.
The good example is [Preconditions](http://guava-libraries.googlecode.com/svn-history/r14/trunk/javadoc/com/google/common/base/Preconditions.html)